### PR TITLE
Add script for installation via curl

### LINKED ISSUE #22: get.sh for installation via curl
## Expected Behaviour

We should write/create a multi-arch get.sh for installation via curl

```
curl URL | sudo sh
```

This would mirror the approach taken with the OpenFaaS CLI. https://github.com/openfaas/cli.openfaas.com/blob/master/get.sh

## Current Behaviour

Download from releases page

### PR DESCRIPTION
**Pull request by larson004**
Closes #22 

## Checklist:

I have:

- [x] updated the documentation and/or roadmap (if required)
- [x] read the [CONTRIBUTION](https://github.com/alexellis/inlets/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: get.sh [hunk 1]
```diff
@@ -0,0 +1,111 @@
+version=$(curl -sI https://github.com/alexellis/inlets/releases/latest | grep Location | awk -F"/" '{ printf "%s", $NF }' | tr -d '\r')
```
Comment (alexellis) [inline get.sh]: Please add a comment/note saying where this is copied from and adapted from (a URL would be good)

File: README.md [hunk 2]
```diff
@@ -54,12 +54,17 @@ Unlike HTTP 1.1 which follows a synchronous request/response model websockets us
 * ~~There is no timeout for when the tunnel is disconnected~~ timeout can be configured via args on the server
 * ~~The upstream URL has to be configured on both server and client until a discovery or service advertisement mechanism is added~~ advertise on the client
 
+### Installing Inlets
```
Comment (alexellis) [inline README.md]: How about:

```
### Get `inlets`
```

**Comment by alexellis** [comment]
This looks great, please see my note and review comments.

Alex

**Comment by alexellis** [comment]
Hi, please could you add your full name with the following?

```
git config user.name "Full name"
git commit --amend --sign-off
git push --force
```

**Comment by larson004** [comment]
@alexellis Done

**Comment by alexellis** [comment]
Merged, thank you. Now do we buy a short domain?

**Pull request merged**